dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -208,6 +212,32 @@ public void run() {
                     multiTreesMap.get(personalization), Optional.of(language.getId()),
                     variantName);
         }
+
+        // MultiTrees as a flattened list
+        final List<MultiTree> savedMultiTrees = multiTreesMap.values().stream()
+                .flatMap(List::stream)
+                .collect(Collectors.toList());
+
+        // Response with container, contentlet and styleProperties
+        return savedContent(savedMultiTrees);
+    }
+
+    /**
+     * Returns a list of the saved MultiTrees with the containerId, uuid, contentletId and styleProperties.
+     * @param savedMultiTrees The list of saved MultiTrees.
+     * @return A list of the saved MultiTrees with the containerId, uuid, contentletId and styleProperties.
+     */
+    private List<Map<String, Object>> savedContent(List<MultiTree> savedMultiTrees) {
```

Comment:
This name made me think that the purpose of the method was saving. Perhaps we need to find a better name here